daemon/config/README.md

```diff
@@ -82,21 +82,33 @@ Please note that your data history will be lost if you have modified `history` p
+|                   update every                   |    `1`     | The frequency in seconds, for data collection. For more information see the [performance guide](/docs/guides/configure/performance.md). These metrics stored as _Tier 0_ data. Explore the tiering mechanism in the [dbengine's reference](/database/engine/README.md#tiering).                                                                                                                                                                                                                                                                                                                                                     |
+|                  storage tiers                   |    `1`     | The number of storage tiers you want to have in your dbengine. Check the tiering mechanism in the [dbengine's reference](/database/engine/README.md#tiering). You can have up to 5 tiers of data (including the _Tier 0_). This number ranges between 1 and 5.                                                                                                                                                                                                                                                                                                                                                                      |
+|           dbengine page cache size MB            |    `32`    | Determines the amount of RAM in MiB that is dedicated to caching for _Tier 0_ Netdata metric values.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+|   dbengine tier **_`Xi`_** page cache size MB    |    `32`    | Determines the amount of RAM in MiB that is dedicated for caching Netdata metric values of the **_`Xi`_** tier. <br /> `Xi belongs to [1..4]`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       ||
```

Comment:
> Xi
> Xi belongs to [1..4]

Why two letters? Can we just use N

> dbengine tier **_`N`_** page cache size MB

If not I suggest using `Xn`


<details>
<summary> Xi is a famous person 😄 </summary>

<img width="1690" alt="Screenshot 2022-07-08 at 15 11 18" src="https://user-images.githubusercontent.com/22274335/177989711-799af237-126f-4163-896c-85548b24d020.png">

</details>